packages/api/src/command/medical/patient/update-patient.ts

```diff
@@ -29,6 +30,10 @@ export const updatePatient = async (patientUpdate: PatientUpdateCmd): Promise<Pa
       personalIdentifiers: sanitized.personalIdentifiers,
       address: sanitized.address,
       contact: sanitized.contact,
+      cxDocumentRequestMetadata:
+        cxDocumentRequestMetadata !== undefined ? cxDocumentRequestMetadata : undefined,
+      cxConsolidatedRequestMetadata:
+        cxConsolidatedRequestMetadata !== undefined ? cxConsolidatedRequestMetadata : undefined,
```

Comment:
The idea here @leite08 is to reset the payload to be empty if they don't pass in metadata on successive requests. Otherwise, if they pass in metadata on one webhook request and then nothing on subsequent requests. We can remove that from here but that logic ought to be added to storeQueryInit()

Reply:
What about the fact `updateQuery` is a generic function and can be called in other use cases/contexts? For example, its used on `PUT /patient/:id` and on `POST /update-all`. These would reset the metadata of a doc query that's in progress, for example.

Reply:
This doesn't need to be changed anymore, right?